influxdb/client.py

```diff
@@ -98,6 +100,13 @@ def __init__(self,
         self._database = database
         self._timeout = timeout
         self._retries = retries
+        if compression is True:
```

Comment:
This is confusing as we don't know what type the `compression` parameter should be. Can you please update the function to include comments on what to expect from this paramater? Should `self._compression` be an int, bool, something else?

Reply:
Well, actually it might be int or bool at that point. If it just True the code will choose a default int value. Seemed sensible to me back when I implemented it. Now I am not so sure anymore ... Suggestion?

Reply:
I think if anything you should define two vars, one for the switch on/off of compression and another for the level.